app/src/main/java/com/parishod/watomatic/NotificationService.java

```diff
@@ -88,6 +94,42 @@ private void sendReply(StatusBarNotification sbn) {
         }
     }
 
+    private static NotificationWear extractQuickReplyNotification(StatusBarNotification sbn) {
+        Notification notification = sbn.getNotification();
+        List<NotificationCompat.Action> actions = new ArrayList<>();
+        for(int i = 0; i < NotificationCompat.getActionCount(notification); i++) {
+            NotificationCompat.Action action = NotificationCompat.getAction(notification, i);
+            actions.add(action);
+        }
+        List<RemoteInput> remoteInputs = new ArrayList<>();
+        PendingIntent pendingIntent = null;
+        for(NotificationCompat.Action act : actions) {
+            if(act != null && act.getRemoteInputs() != null
+                && remoteInputs.size() == 0 ) {
+                for(int x = 0; x < act.getRemoteInputs().length; x++) {
+                    RemoteInput remoteInput = act.getRemoteInputs()[x];
+                    if((remoteInput.getResultKey().toLowerCase().contains("reply")
```

Comment:
Using English word "reply" may not work for other languages

Reply:
Well actually i realised we dont need to check "reply" as remoteInput object only will for reply